app/models/user.rb

```diff
@@ -6,4 +6,8 @@ class User < ApplicationRecord
   has_secure_password
 
   has_many :characters
+
+  def admin?
+    self.status == "admin"
```

Comment:
it's preferable to use an enum for a 'role' like admin, or a boolean, as opposed to storing strings. Enum strings get stored one time as opposed to storing a string over and over in every row of the table, and enums (which are now integers) are way faster to search in a database.

Reply:
yeah painted myself into a corner here, will fix in a later issue